packages/custom-url/tests/Application/config/webspaces/sulu.io.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.1.xsd">
+
+    <name>Sulu CMF</name>
+    <key>sulu_io</key>
+
+    <security>
+        <system>Website</system>
+    </security>
+
+    <localizations>
+        <localization language="de" default="true"/>
+        <localization language="en"/>
+    </localizations>
+
+    <theme>default</theme>
+
+    <default-templates>
+        <default-template type="homepage">default</default-template>
+        <default-template type="page">default</default-template>
+    </default-templates>
+
+    <templates>
+        <template type="search">search/search</template>
+    </templates>
+
+    <navigation>
+        <contexts>
+            <context key="main">
+                <meta>
+                    <title lang="en">Mainnavigation</title>
+                </meta>
+            </context>
+        </contexts>
+    </navigation>
+
+    <portals>
+        <portal>
+            <name>Sulu CMF AT</name>
+            <key>sulucmf_at</key>
+
+            <environments>
+                <environment type="prod">
+                    <urls>
+                        <url language="de">{host}</url>
+                        <url language="en">{host}/en</url>
```

Comment:
didnt you need a custom-url definition to have functional testcases?

Reply:
Technically only for rendering the admin form and that's not being tested. We're just testing the API to get and update the entries. But we might also want to check the admin configuration endpoint for the custom urls to check there are no bc breaks there.